frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -142,32 +142,37 @@ const ModelDetailsPage: React.FC = conditionalArea(
         </Breadcrumb>
       }
       title={
-        <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
-          <FlexItem>
-            <BrandImage src={model?.logo ?? ''} alt="" />
-          </FlexItem>
-          <FlexItem>
-            <Stack>
+        <Flex spaceItems={{ default: 'spaceItemsMd' }} alignItems={{ default: 'alignItemsCenter' }}>
+          {model?.logo ? (
+            <img src={model.logo} alt="model logo" style={{ height: '40px', width: '40px' }} />
+          ) : (
+            <Skeleton
+              shape="square"
+              width="40px"
+              height="40px"
+              screenreaderText="Brand image loading"
+            />
+          )}
```

Comment:
We may want to factor this into a small component later since it's duplicated, but that doesn't need to block us